src/modules/flow/twitter/twitter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "twitter",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD-3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "twitter/client",
+      "category": "output/network",
+      "description": "A twitter's node client",
+      "methods": {
+        "close": "twitter_close",
+        "open": "twitter_open"
+      },
+      "options": {
+	"members": [
+	  {
+	    "data_type": "string",
+	    "description": "The consumer key (application's token)",
+	    "name": "consumer_key"
+	  },
+	  {
+	    "data_type": "string",
+	    "description": "The consumer secret key (application's secret token)",
+	    "name": "consumer_secret"
+	  }
+	],
+	"version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The token returned on twitter's oauth process",
+          "methods": {
+            "process": "token_process"
+          },
+          "name": "TOKEN"
+        },
+	{
+          "data_type": "string",
+          "description": "Post the string that comes into twitter",
+          "methods": {
+            "process": "post_process"
+          },
+          "name": "TWEET"
```

Comment:
you liked it because you did a very simplistic version of only 2 nodes. This doesn't scale.

If we come back to this module to add all supported API version, then what we're going to do? Increase its version and add ports like crazy? :-D

It's better to reuse the C functions but provide specific nodes that allow using Twitter's API, like I said tweet could get a location, etc.